New package: PyBoltz v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: PyBoltz
- Repository: https://github.com/MurrellGroup/PyBoltz.jl
- Created by: @AntonOresten
- Version: v0.1.0
- Commit: f317a381751a5a98bf1b0885070644a17ddd7284
- Reviewed by: @AntonOresten
- Reference: https://github.com/MurrellGroup/PyBoltz.jl/commit/f317a381751a5a98bf1b0885070644a17ddd7284#commitcomment-157019405
- Description: Julia bindings for Boltz-1
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
- Add exported `predict` function.
- Add public `PyBoltz.Schema` submodule.
`````
<!-- END RELEASE NOTES -->

<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6a20a55b5f6c7ac33a1a738ba3d8ddcaae73830caf71092ff4859e57057c74c1f8c03a0e07d44ed1fcc4f04c5c9a33351c2e53541a26682f6821e76b835b9d70c94c77a7b92174dd15c96a27f4b5956dc97cd01ba13e357446956e31b08d211a150a480927f361984d61e04055f78070e993f16d8f3e632f94c4f26337cd8dda7cd4265ad30e21fa24c297f10bfeea44ba1f98538f2da620500dfda3409cac2f6575b1f5fb7962e56a8394427e1b7e0dc -->

AntonOresten commented:
[noblock]
See #130868 and https://julialang.slack.com/archives/C6M4DQA5P/p1747166341838319

JuliaRegistrator committed:
New package: PyBoltz v0.1.0

UUID: 0bca4874-a2b8-4dca-bc03-d2d86e344f1f
Repo: https://github.com/MurrellGroup/PyBoltz.jl.git
Tree: 81b4ae1a544da749a4e310084a91d11396eb315f

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
